test/functional/test_framework/test_node.py

```diff
@@ -307,21 +307,31 @@ def wait_until_stopped(self, timeout=BITCOIND_PROC_WAIT_TIMEOUT):
         wait_until(self.is_node_stopped, timeout=timeout)
 
     @contextlib.contextmanager
-    def assert_debug_log(self, expected_msgs):
+    def assert_debug_log(self, expected_msgs, timeout=2):
+        time_end = time.time() + timeout
+        print_log = ""
```

Comment:
no need for this. python doesn't have scope blocks

Reply:
Fixed